doc/source/serve/tutorials/triton-server-integration.md

```diff
@@ -0,0 +1,72 @@
+(serve-triton-server-integration)=
+
+# Serving a Resenet model with Triton Server
+This guide shows how to serve models with [NVIDIA Triton Server](https://github.com/triton-inference-server/server) using Ray Serve.
+
+## Installation
+Here is the Dockerfile example for installing Triton Server with Ray Serve.
+
+## Inference Code
+Before starting the triton serve to do the inference, you need to place your model into the model repository (e.g. `/tmp/models`). So that the triton serve can get access to your models.
+You can follow the [guide][https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_repository.md#model-repository] to setup the model repo.
+
+In the example, the model repo is under `/tmp/models` and the model name is `resnet50_libtorch`.
+
+```bash
+ray@ip-10-0-60-123:~/default/tutorials/RayServe/models_test$ ls /tmp/models/resnet50_libtorch/
+1  config.pbtxt  resnet50_labels.txt
+```
+
+Here is the inference code example for serving a model with Triton Server.
+
+```python
+import numpy
+import requests
+from fastapi import FastAPI
+from ray import serve
+from tritonserver_api import TritonServer
+
+
+# 1: Define a FastAPI app and wrap it in a deployment with a route handler.
+app = FastAPI()
+
+
+@serve.deployment(route_prefix="/", ray_actor_options={"num_gpus": 1})
```

Comment:
can we add some autoscaling options?

Reply:
added, just finish testing, it is working normally.